frontends/parsers/v1/v1parser.ypp

```diff
@@ -485,12 +494,14 @@ case_value:
       { $$ = CaseValue($1 ? $1 : new IR::Constant(-1), new IR::Constant(-1)); }
     | const_expression MASK const_expression
       { $$ = CaseValue($1 ? $1 : new IR::Constant(-1), $3 ? $3 : new IR::Constant(0)); }
+    | name
+      { $$ = CaseValue(new IR::StringLiteral(@1, $1), new IR::Constant(-1)); }
```

Comment:
This should be an identifier, not a string literal.

Reply:
Actually probably should be a PathExpression